tools-v2/internal/utils/gap.go

```diff
@@ -110,9 +110,9 @@ func NewCopysetLeaderInfo() *CopysetLeaderInfo {
 // snapshot_status: IDLE
 // replicator_3298534883333@10.219.192.50:6800:0: next_index=4  flying_append_entries_size=0 idle hc=150785 ac=2 ic=0
 // replicator_2203318222854@10.219.192.50:6802:0: next_index=4  flying_append_entries_size=0 idle hc=150785 ac=2 ic=0
-func GetLeaderCopysetGap(addr string, key2LeaderInfo *sync.Map, timeout time.Duration) *cmderror.CmdError {
-	metric := basecmd.NewMetric([]string{addr}, RAFT_STAT_URL, timeout)
-	result, _ := basecmd.QueryMetric(metric)
+func GetLeaderCopysetGap(addr string, timeout time.Duration) (CopysetLeaderInfo, *cmderror.CmdError) {
```

Comment:
Wouldn't it be possible to obtain only one modification at a time in this way, and to query multiple times?